core/src/main/java/com/linecorp/armeria/server/docs/DocService.java

```diff
@@ -108,14 +110,17 @@ public static DocServiceBuilder builder() {
      * Creates a new instance.
      */
     public DocService() {
-        this(ImmutableMap.of(), ImmutableMap.of(), ImmutableList.of(), DocServiceBuilder.ALL_SERVICES);
+        this(ImmutableMap.of(), ImmutableMap.of(), ImmutableMap.of(),
```

Comment:
This was already a bit crazy but now even more so probably needs param comments e.g., `/* exampleHttpHeaders= */ ImmutableMap.of()`

Reply:
Let me add param comments.🤣🙈